### PR TITLE
remove logitswarper

### PR DESCRIPTION
**Pull request by grimoire**
Latest transformers has removed logitswarper

### CODE REVIEW

**Comment by lvhan028** [comment]
Does it affect the customized logits processor function?

**Comment by grimoire** [comment]
custom logits is treated as function call.

**Review by AllentDan — APPROVED** [review]
LGTM

**Comment by grimoire** [comment]
> Does it work for lower-version transformers?

Yes
